owsi-core/owsi-core-components/owsi-core-component-jpa-more/src/main/java/fr/openwide/core/jpa/more/business/difference/util/IHistoryDifferenceGenerator.java

```diff
@@ -0,0 +1,14 @@
+package fr.openwide.core.jpa.more.business.difference.util;
+
+import java.util.List;
+
+import com.google.common.base.Supplier;
+
+import fr.openwide.core.jpa.more.business.difference.model.Difference;
+import fr.openwide.core.jpa.more.business.history.model.AbstractHistoryDifference;
+
+public interface IHistoryDifferenceGenerator<T> {
```

Comment:
I'd prefer sticking with this name so that there's no confusion possible with the Converter infrastructure.